pyanaconda/modules/common/task/task.py

```diff
@@ -105,7 +105,7 @@ def _task_run_callback(self):
             log.info("'%s' is canceled.", self.name)
             return
 
-        self.report_progress(self.name, step_number=1)
+        log.info(self.name)
```

Comment:
I am wondering, what will that do? Just a random "Setting up disks" in the logs?

It's not really important, though.

Reply:
I think it provides a nice context by default. I will keep it there for now. We can always changed it.

![name](https://user-images.githubusercontent.com/7396092/155523767-fac10140-b0fe-479e-b2aa-418edf2ce5c5.png)